src/transformers/models/recurrent_gemma/modeling_recurrent_gemma.py

```diff
@@ -895,37 +896,31 @@ def forward(
             hidden_states=outputs.hidden_states,
         )
 
-    # Ignore copy
+    # Ignore copy (no past_key_values, no inputs_embeds)
```

Comment:
`inputs_embeds` was hitting corner cases, removed the support for now

Reply:
mmm this is breaking can't we keep input_embeds?